corehq/apps/users/migrations/0033_rename_sqluserrole.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.24 on 2021-08-02 14:42
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0032_add_download_reports_permission'),
+    ]
+
+    operations = [
+        # nothing is changing in the DB but Django will remove and re-create all constraints
```

Comment:
Is it valuable to do this? Asking because I haven't done it in previous migrations.

Reply:
I didn't expect this migration to produce and SQL since it's only the model name that is changing and not the DB table but for some reason Django wanted to remove all the constraints and add them back again with the same names. Since it wasn't necessary to do that I decided to take this approach.